assets/js/app.js

```diff
@@ -40,10 +41,11 @@ $(() => {
   // Set up Handlebars.js
   Handlebars.registerHelper('isArray', isArray)
   Handlebars.registerHelper('isLocal', isLocal)
-  Handlebars.registerHelper('isNonEmptyArray', isNonEmptyArray)
   Handlebars.registerHelper('groupChanged', groupChanged)
+  Handlebars.registerHelper('isNonEmptyArray', isNonEmptyArray)
```

Comment:
This should be moved where it was (alphabetic order).

Reply:
I have already addressed this issue.